Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -2314,11 +2345,45 @@ void TextureCacheBase::CopyRenderTargetToTexture(
 
   // Even if the copy is deferred, still compute the hash. This way if the copy is used as a texture
   // in a subsequent draw before it is flushed, it will have the same hash.
-  if (entry)
+  if (entry == nullptr)
+    return;
+
+  const u64 hash = entry->CalculateHash();
+  entry->SetHashes(hash, hash);
+  textures_by_address.emplace(dstAddr, entry);
+
+  if (is_xfb_copy)
+  {
+    if (!g_ActiveConfig.bDumpXFBTarget && !g_ActiveConfig.bGraphicMods)
+      return;
+    const std::string id = fmt::format("{}x{}_{:016x}", tex_w, tex_h, hash);
```

Comment:
Hashes are of very dubious usefulness for xfb copies.

When we are doing xfb to texture, we zero out the memory backing the xfb copy, so all you are hashing is `tex_w * tex_h` zeros. Which makes the hash is just a more obfuscated way of specifying tex_w and tex_h, which are already part of the your identifier.  

When we are doing xfb to ram (without defer), the hash is based on the xfb contents, which change based on the user's graphics settings and gpu drivers. It's simply not stable enough to be useful.

Reply:
Thanks for catching this phire.  You're correct on all accounts.  Embarassing because I should have remembered this working on hybrid xfb!

What I wanted was the hash of the image.  But then your point stands that the hash will change based on resolution.

I will have to rethink this approach for targeting FMVs.  I was a little hasty here.

I'll remove this functionality for now.

Reply:
I have vague memories of trying to make xfb uploads act as much like a regular texture as possible, just with a fake YUV 4:2:2 texture format. 

If that worked, they should just show up as regular textures.